provider/ec2/local_test.go

```diff
@@ -1566,7 +1566,7 @@ func (t *localServerSuite) TestValidateImageMetadata(c *gc.C) {
 	image_ids, _, err := imagemetadata.ValidateImageMetadata(params)
 	c.Assert(err, jc.ErrorIsNil)
 	sort.Strings(image_ids)
-	c.Assert(image_ids, gc.DeepEquals, []string{"ami-00000133", "ami-00000135", "ami-00000139"})
+	c.Assert(image_ids, gc.DeepEquals, []string{"ami-00001133", "ami-00001135", "ami-00001139"})
```

Comment:
what's this change?

Reply:
This was using mock AMIs and I just wanted to highlight that bionic had a unique AMI code for the test. "ami-00000133" vs "ami-00001133".